src/widgets/_panes.scss

```diff
@@ -12,13 +12,11 @@ paned.horizontal > separator {
         background-image:
             linear-gradient(
                 to right,
-                #{'@menu_separator'} 1px,
-                rgba(white, 0.01) 1px,
-                rgba(white, 0.01) 2px,
```

Comment:
This is never shown because of the way z-index works in gtk. The pane is under the view next to it